tests/conftest.py

```diff
@@ -155,11 +162,28 @@ def id_token():
     )
 
 
+@pytest.fixture
+def refresh_token():
+    return "test_refresh_token"
+
+
 @pytest.fixture(autouse=False)
-def token_response(mocker, access_token, id_token):
+def token_response(mocker, access_token, id_token, refresh_token):
     mocker.patch(
         "flask_cognito_lib.plugin.CognitoAuth.get_tokens",
-        return_value=CognitoTokenResponse(access_token=access_token, id_token=id_token),
+        return_value=CognitoTokenResponse(
+            access_token=access_token, id_token=id_token, refresh_token=refresh_token
```

Comment:
```suggestion
            access_token=access_token, id_token=id_token, refresh_token=refresh_token,
```

Adding a trailing comma at the end here should make `black` put each keyword arg on a new line which might read a bit easier